src/components/ScheduleSwitcher.tsx

```
@@ -61,13 +61,26 @@ function SelectWithWarn(props: {
   const { state, saveId, saves } = props;
   const [confirmSave, setConfirmSave] = useState("");
   const confirmName = saves.find((save) => save.id === confirmSave)?.name;
+  const defaultScheduleId = state.getDefaultSchedule();
+
+  const formatScheduleName = (id: string, name: string) => {
```

This shows schedule 1 as the default, even when none is set. This could get confusing cause it also allows you to set it as the default which doesnt change anything? Maybe best to just hide (default) if none is selected